src/components/organisms/FilterBar.tsx

```diff
@@ -62,9 +62,15 @@ const FilterBar: FC<IProps> = () => {
                   </Button.Standard>
                 </Grid>
               </Grid>
-              <Grid item>
-                <Box mt={1}>{isDescOpen && <Typography.Body1>{store.activeNewsFlash?.title}</Typography.Body1>}</Box>
-              </Grid>
+              {isDescOpen && (
```

Comment:
I though it would make more sense to render the entire box conditionally over an empty element when the description is toggled off. If we use reference it in any other way, I can move the condition back inside.